.swiftlint.yml

```diff
@@ -13,6 +13,6 @@ private_over_fileprivate:
   validate_extensions: true
 
 file_length:
-  warning: 450
+  warning: 485
```

Comment:
Isn't more dangerous disable the rule for the whole file? That way we will completely forget that file and it may increase forever. Keeping it enabled, even if had to tweak the parameters once in a while, will make us ponder if it's time to split the file or not.

I hope this get implemented soon https://github.com/realm/SwiftLint/issues/800

Reply:
Honestly, I don't care so much about file length. I care more about keeping similar structures together so that files are contextual. Specially in languages where we can't easily create subfolders to represent a module.

Anyway, I's a mater of personal preference at this point. I won't block the PR because if this.